wix-stores-backend/wix-stores-backend/guides/eCommerce Integration.md

```diff
@@ -45,3 +45,57 @@ When the inventory of a product's variants is not managed (`product.manageVarian
   }
 }
 ```
+
+## Velo Code Example
+
+The example below uses frontend code on the Stores Product Page and performs the following actions:
++ Gets the info of the product currently displayed in the Product Page
```

Comment:
- info -> information
- Should the bullet points have a period at the end? Not sure.